test/load.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
```

Comment:
is n't this too much for bash? I mean wouldn't this be quite simplified if written in golang?

@observatorium/maintainers what do you think?